<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0" xmlns="http://www.qt-project.org/ns/ui">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTSP Stream Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="videoPathLineEdit">
      <property name="placeholderText">
       <string>Введите путь к видеофайлу...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="selectFileButton">
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startServerButton">
      <property name="text">
       <string>Запустить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
